keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -61,9 +57,9 @@ class ResourceController(
     produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
   @ResponseStatus(CREATED)
-  fun create(@RequestBody submittedResource: SubmittedResource<*>): Resource<*> {
+  fun create(@RequestBody submittedResource: SubmittedResource<Any>): Resource<out Any> {
     log.debug("Creating: $submittedResource")
-    return resourcePersister.handle(ResourceCreated(submittedResource))
+    return resourcePersister.create(submittedResource)
```

Comment:
This `handle(some type of event)` formation is cruft from when `ResourcePersister` was consuming events from the Spring event bus. It makes more sense to have regular ol' individual methods now.